src/main/java/calculator/StringPreprocessor.java

```diff
@@ -0,0 +1,18 @@
+package calculator;
+
+public class StringPreprocessor {
+    private static final String CUSTOM_DELIMITER_START = "//";
+    private static final String CUSTOM_DELIMITER_END = "\\n";
+
+    public String removeDelimiterPart(String input) {
+        if (input == null || input.isEmpty()) {
```

Comment:
String.isBlank()를 사용하시면 공백도 검증할 수 있습니다!
```suggestion
        if (input == null || input.isBlank()) {
```

Reply:
isBlank는 몰랐습니다! 자바 문법도 열심히 공부해봐야겠군요.. 감사합니다!